### PR TITLE
You can no longer resist grabs while handcuffed

### PR DESCRIPTION
**Pull request by Garen7**
## About The Pull Request
<details>
<summary>
	Now you have to break out of the cuffs before you can resist out of the grab just like in every other ss13 server.
</summary>
<hr>
	I feel like this is a bug but if I'm wrong someone correct me, either way its pretty annoying when the person you arrested keeps resisting out while you try to bring them to brig, strip them, or whatever you were doing with them. 
<hr>
</details>

## Changelog
:cl:
fix: You can no longer resist grabs while handcuffed
/:cl:

### CODE REVIEW

**Comment by Garen7** [comment]
Not sure why the map diff bot is failing since I don't have any changes to any maps.

File: code/modules/mob/living/carbon/resist.dm [hunk 1]
```diff
@@ -58,6 +58,9 @@
 
 
 /mob/living/proc/resist_grab()
+	var/mob/living/carbon/C = src
```
Comment (mattroks101) [inline code/modules/mob/living/carbon/resist.dm]: You should probably be checking if they're a carbon before you cast them as one.

**Comment by Garen7** [comment]
Sorry I’ve been kinda busy I should be able to edit it today

File: code/modules/mob/living/carbon/resist.dm (hunk 1 above)
Comment (Garen7) [reply code/modules/mob/living/carbon/resist.dm]: Are you sure? Most people type check by doing that and the [byond reference](http://www.byond.com/docs/ref/#/proc/istype) says it’s ok so I’m not sure if there’s a dog difference

Comment (Garen7) [reply code/modules/mob/living/carbon/resist.dm]: @mattroks101 hey do you want me to change it to check if its a carbon before I cast it or is it good as is.

File: code/modules/mob/living/carbon/resist.dm [hunk 2]
```diff
@@ -58,6 +58,9 @@
...
+	if(istype(C) && C.handcuffed)
+		return
```
Comment (LethalGhost) [inline code/modules/mob/living/carbon/resist.dm]:
```suggestion
/mob/living/carbon/resist_grab()
	return !handcuffed && ..()
```

File: code/modules/mob/living/carbon/resist.dm (hunk 1 above)
Comment (LethalGhost) [reply code/modules/mob/living/carbon/resist.dm]: @Garen7 
How about
```
/mob/living/carbon/resist_grab()
	return !handcuffed && ..()
```

Comment (Garen7) [reply code/modules/mob/living/carbon/resist.dm]: @LethalGhost 👍

**Pull request merged**